inc/maze.h

```diff
@@ -21,29 +21,127 @@
 # define MAZE_E 4
 # define MAZE_W 8
 
+/**
+ * @brief Magic number from RAND_MAX
+ */
 # define FT_RAND_MAX 32768
 
-typedef struct s_posvec
+typedef struct s_posvec	t_posvec;
```

Comment:
Yes I get that splitting typedefs and struct defitions are technichally "more readable", but this is a little unreasonable since there is only one structure in this header.